cohttp_lwt_jsoo_test/src/test.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const assert = require('assert');
+
+global.XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+const tests = require('./cohttp_lwt_jsoo_test.bc.js')
+
+async function main() {
+  {
+    const [status, body] = await tests.request("https://mirage.io");
```

Comment:
We may want to spin our own local http server to avoid depending on external connectivity.

Reply:
We could recycle https://github.com/mirage/ocaml-cohttp/blob/master/cohttp_lwt_unix_test/src/cohttp_lwt_unix_test.ml#L27 or some other example from the code for it, like https://github.com/mirage/ocaml-cohttp/blob/master/examples/lwt_unix_doc/server_lwt.ml